app/models/event_topic.py

```diff
@@ -20,6 +20,10 @@ class EventTopic(db.Model):
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     name = db.Column(db.String, nullable=False)
+    system_image_url = db.Column(
+        db.String, nullable=False,
+        default='http://www.cih.org/resources/images/event_header.jpg'
```

Comment:
This resource should be generic like the empty avatar when you create a new account

Reply:
@iamareebjamal 
Do you mean this 
`default=''`

Reply:
No I mean for avatar images like this are used - http://hotchillitri.co.uk/wp-content/uploads/2016/10/empty-avatar.jpg

Similarly such generic image should be used with appropriate creative commons license

Reply:
How will this work http://vitriviera.com/static/media/pre.17254662.jpg

Reply:
It should be generic. Remember how you use a service and expect it to have sane defaults for you

Example - https://github.com/fossasia/open-event-orga-app/blob/development/app/src/main/res/drawable/header.png

Reply:
and please download the image. Otherwise the project depends on an external URL that might change

Reply:
I have uploaded that image at https://image.ibb.co/iTrd5y/system_image_common.png 
Which URL will not change 
Will it be okay @maxlorenz @iamareebjamal

Reply:
This is based on external service which can be down without any notice